gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -501,6 +502,14 @@ public static Builder builder() {
   BlobInfo create(BlobInfo blobInfo, byte[] content, BlobTargetOption... options);
 
   /**
+   * Create a new blob.
```

Comment:
Can we update the javadoc in this PR and recommend using the `writer` method for large content as it uses resumable upload?